tkg/client/machine_deployment_cc.go

```diff
@@ -24,6 +24,23 @@ func DoSetMachineDeploymentCC(clusterClient clusterclient.Client, cluster *capi.
 		return errors.New("cluster topology workers are not set. please repair your cluster before trying again")
 	}
 
+	var nodePoolVarFound bool
+	for i := range cluster.Spec.Topology.Variables {
+		if cluster.Spec.Topology.Variables[i].Name == "nodePoolLabels" {
+			nodePoolVarFound = true
+		}
+	}
```

Comment:
We helpers to make it easier to read and write cluster variables: see "github.com/vmware-tanzu/tanzu-framework/util/topology" package

Reply:
The util function in that package is for ClusterClass. I'm operating directly on a Cluster, should I update the utils to include a similar function for Clusters?